### ISSUE TITLE
Feature request: NVDA doesn't read items in Sam Broadcaster's list views

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3262]
**Reported by oaron on 2013-06-02 21:53**
I'm using the Latest NVDA snapshot, and Sam Broadcaster V 4.7.1, The users can broadcast in internet radio with this software. In the list views, the chosen songs appear, and they can move on with the arrow keys. I can read the items with the object navigation, but when I use arrow keys, NVDA doesn't announce anything.
I tried to create an appmodule (and Mesar helped me), but it doesn't work. Do you have any idea what the problem is, ,how can I fix it?

**Comment by bhavyashah** [comment]
I am under the impression that @josephsl was experimenting with SAM Broadcaster and even designed a 'skeleton' app module for the same. If this impression was an illusion, please excuse me, but if not, could you please test for the reported issue and kindly share your findings? While the version number system for SAM was probably different four years ago (when this ticket was originally filed), the latest stable release is 2017.6 released this July, implying that it is very much possible that an NVDA update and more likely a SAM Broadcaster update may have resolved this bug.

**Comment by josephsl** [comment]
Hi, after playing with it, I came to the conclusion that it is better to let the app developer take care of this. Thanks.

**Comment by ehollig** [comment]
@josephsl, are you happy with this issue to be closed?

**Comment by josephsl** [comment]
Hi, yes.  Thanks.

 ----- Original Message -----

**Comment by ehollig** [comment]
Closing
